app/frontend/src/features/profile/actions/PendingFriendRequestSent.tsx

```diff
@@ -0,0 +1,25 @@
+import { makeStyles } from "@material-ui/core";
+import classNames from "classnames";
+import Button from "components/Button";
+import { PersonAddIcon } from "components/Icons";
+import { PENDING_REQUEST_SENT } from "features/connections/constants";
+import React from "react";
+
+const useStyles = makeStyles((theme) => ({
+  disabledButton: {
+    backgroundColor: theme.palette.grey[100],
+  },
+}));
+
+export default function PendingFriendRequestSent() {
+  const classes = useStyles();
+  return (
+    <Button
+      startIcon={<PersonAddIcon />}
+      className={classNames({ [classes.disabledButton]: true })}
```

Comment:
`classNames(classes.disabledButton)` works. But if there's only one class, just use the class directly: 
```suggestion
      className={classes.disabledButton}
```

Reply:
However, I'm surprised this class is necessary - shouldn't it be appropriately styled anyway if `disabled={true}`?

Reply:
You're right, I had copy pasted from previous code but indeed the class is not necessary! It styles anyways with `disabled={true}`